src/webgpu/api/validation/encoding/cmds/render/dynamic_state.spec.ts

```diff
@@ -130,71 +130,97 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
```

Comment:
If you want to can change this to

```suggestion
export const g = makeTestGroup(MixLimitsTestMixin(F));
```

And it will test with the adaptor's max limits

Reply:
Done